pybamm/solvers/casadi_solver.py

```diff
@@ -71,6 +71,7 @@ def __init__(
         root_tol=1e-6,
         max_step_decrease_count=5,
         dt_max=None,
+        extrap_tol=1e-3,
```

Comment:
can you add this to the docstring? also, do other solvers have this problem with interpolants (i.e. they fail without giving much info)?

Reply:
Done